src/components/util/Themes.ts

```diff
@@ -0,0 +1,9 @@
+export function isHalloween() {
+	const dateToday = new Date();
+
+	if ((dateToday.getMonth() === 9 && dateToday.getDate() >= 30) || (dateToday.getMonth() === 10 && dateToday.getDate() === 1)) {
```

Comment:
```suggestion
	if ((dateToday.getMonth() === 9 && dateToday.getDate() >= 29) || (dateToday.getMonth() === 10 && dateToday.getDate() === 1)) {
```

Sorry, the event time changed so it's the 29th now!